Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+
+namespace BootCamp.Chapter
+{
+    class Lesson3
+    {
+        public static float CalculateBmi(float height, float weight)
+        {
+            // Calculate BMI
+            // Formula: weight (kg) / [height (m)]**2
+
+            //float heightM = height / 100;  // calculate height in meters
+
+            float BMI = weight / (height * height);
```

Comment:
This should be all lowercase to fit with the rest of your document. If you violate conventions it will always catch a readers attention. Please change that in order to prevent getting into bad habits.